### PR TITLE
Documentation: Update README.md to reflect plugin split

### PR DESCRIPTION
**Pull request by mcastelino**
Update README.md to call out how to use priv-net-run.sh and
docker-run.sh scripts post plugin split.

Signed-off-by: Manohar Castelino <manohar.r.castelino@intel.com>

### CODE REVIEW

**Comment by wppersian-farsi** [comment]
#469

File: README.md [hunk 1]
```diff
@@ -116,13 +116,14 @@ The directory `/etc/cni/net.d` is the default location in which the scripts will
...
+$ cd $GOPATH/src/github.com/containernetworking/plugins
 $ ./build.sh
 ```
 
 Finally, execute a command (`ifconfig` in this example) in a private network namespace that has joined the `mynet` network:
 
 ```bash
-$ CNI_PATH=`pwd`/bin
+$ CNI_PATH=$GOPATH/src/github.com/containernetworking/plugins/bin
 $ cd scripts
```
Comment (rosenhouse) [inline README.md]: I think this needs to change to
```
cd $GOPATH/src/github.com/containernetworking/cni/scripts
```

The same as what you already did below for the Docker instructions.

Comment (mcastelino) [inline README.md]: Missed that. Updated the PR to fix it.

**Comment by wppersian-farsi** [comment]
@

**Pull request merged**